MSALiOSB2C/ViewController.swift

```diff
@@ -226,25 +173,33 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
              authentication flow will be locked down to.
              - completionBlock: The completion block that will be called when the authentication
              flow completes, or encounters an error.
+             
+             Note the fact that we also look for `InteractionRequired` as an error code and
+             prompt the user interactively. Often times the inability to use a refresh token
+             is from either a password change, refersh token expiring, or other event that
+             can be remedied by the user signing in again. This shouldn't be necessary at every
+             `AcquireTokenSilent()`. If you are experiencing that in your application, make
+             sure you are using the cache correctly and using the same authority.
              */
             
-            let thisUser = try self.getUserByPolicy(withUsers: application.users(), forPolicy: kSignupOrSigninPolicy)
+            guard let account = self.account else {
```

Comment:
this will only be set after acquire token interactive right? How will this work after application restart?

Reply:
Yes, this is set only after interactive login. Not sure if I understand the question. I don't have solid background on iOS application lifecycle. What is going to happen with the variable after application restart?